nomad/settings.py

```diff
@@ -13,6 +13,14 @@
 import os
 import dj_database_url
 
+# Setting up email server
+from .email_info import EMAIL_HOST, EMAIL_HOST_PASSWORD, EMAIL_HOST_USER, EMAIL_PORT, EMAIL_USE_TLS
+EMAIL_USE_TLS = EMAIL_USE_TLS
```

Comment:
I don't think you need to redefine the variables here again
If you are importing them from some other file, they become a part of this file now

Reply:
i have to otherwise flake8 would keep failing

Reply:
Then declare those variables here itself 
Remove `nomad/email_info.py`